plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -167,39 +158,12 @@ public ElasticsearchTableHandle getTableHandle(ConnectorSession session, SchemaT
             String[] parts = tableName.getTableName().split(":", 2);
             String table = parts[0];
             Optional<String> query = Optional.empty();
-            ElasticsearchTableHandle.Type type = SCAN;
             if (parts.length == 2) {
-                // TODO this query pass-through mechanism is deprecated in favor of the `raw_query` table function.
-                //  it should be eventually removed: https://github.com/trinodb/trino/issues/13050
-                if (table.endsWith(PASSTHROUGH_QUERY_SUFFIX)) {
```

Comment:
After we remove it, in a query like:
`SELECT * FROM es.default."<index>$query:<es-query>"`,
the whole chunk: `<index>$query` will be interpreted as the index name. Perhaps this is a valid index name. However, for the case when the index is not found, and someone was just trying to use the old pass-through mechanism, we might try to produce an error message with a hint.